pallets/ah-migrator/src/multisig.rs

```diff
@@ -0,0 +1,147 @@
+// This file is part of Substrate.
+
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use crate::*;
+use hex_literal::hex;
+
+#[cfg(feature = "std")]
+use pallet_rc_migrator::types::AccountIdOf;
+
+/// These multisigs have historical issues where the deposit is missing for the creator.
+const KNOWN_BAD_MULTISIGS: &[AccountId32] = &[
```

Comment:
This code cannot NOT be feature gated :D are these problematic in Polkadot or Kusama?